docs/BR.md

```diff
@@ -1363,11 +1498,11 @@ Log records MUST include the following elements:
 
 The CA SHALL retain, for at least two years:
 
-  1. CA certificate and key lifecycle management event records (as set forth in Section 5.4.1 (1)) after the later occurrence of:
+  1. CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (1)) after the later occurrence of:
      1. the destruction of the CA Private Key; or
      2. the revocation or expiration of the final CA Certificate in that set of Certificates that have an X.509v3 `basicConstraints` extension with the `cA` field set to true and which share a common Public Key corresponding to the CA Private Key;
-  2. Subscriber Certificate lifecycle management event records (as set forth in Section 5.4.1 (2)) after the revocation or expiration of the Subscriber Certificate;
-  3. Any security event records (as set forth in Section 5.4.1 (3)) after the event occurred.
+  2. Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (2)) after the revocation or expiration of the Subscriber Certificate;
+  3. Any security event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (3)) after the event occurred.
```

Comment:
should **(3)** be replaced with **3.** ?

Reply:
Our historical style, both in and out of the documents, has been to refer to list items in parenthesis. This is much clearer in the EV Guidelines, which does so rather extensively, but note even the preceding line (`(2)`).

The current style is, unsurprisingly, borrowed from legal citation style (e.g. [OSCOLA](https://www.law.ox.ac.uk/sites/files/oxlaw/oscola_4th_edn_hart_2012.pdf), 2.4.2) for referencing subsections and paragraphs, no doubt because the EVGs were primarily drafted by those with legal backgrounds 😅 This carried over through to the BRs.

In that, Sections typically have a distinct indicator/abbreviation (`Section`, `s`/`sect.`, `§`), followed by the further divisions (subsection, paragraph, subparagraph) denoted in parenthesis, which themselves follow consistent rules (e.g. see [Wyoming State Code](https://law.justia.com/codes/wyoming/2014/title-8/chapter-1/section-8-1-105))

Reply:
I think this information should be added to the README.md file or somewhere in the CA/B Forum wiki for others to know :) This writing style is not very popular for technical documents some of us are used to.

At least this reference can set the expectations for consistency throughout the CABF documents, meaning that all references need to follow this style. It's something to consider in a future update.

Reply:
Yes, the is something I’ve raised in the past (e.g. https://archive.cabforum.org/pipermail/infrastructure/2020-March/000192.html )